java/client/src/org/openqa/selenium/By.java

```diff
@@ -44,6 +44,10 @@
  * </code>
  */
 public abstract class By {
+  static {
```

Comment:
@lukeis The problem is, that I cannot see a single place that is guaranteed to be called before any other selenium code. I tried to find few places that cover most of cases.
As I see it, RemoteWebDriver doesn't suite because not all webdrivers extend RemoteWebDriver. Am I right?